examples/jsm/utils/RoughnessMipmapper.d.ts

```diff
@@ -0,0 +1,12 @@
+import {
+	WebGLRenderer,
+    MeshStandardMaterial
+} from '../../../src/Three';
+
+export class RoughnessMipmapper {
+
+	constructor( renderer:WebGLRenderer );
+	generateMipmaps( material:MeshStandardMaterial ): void;
```

Comment:
GLTFLoader currently constructs MeshStandardMaterial, but the docs below mention only MeshPhysicalMaterial – does this apply to both the parent and child class?

Reply:
I believe it applies equally to both; Physical just adds clearcoat, right? The relationship between the normal map and roughness map remains the same, especially since there is no separate clearcoat roughness map.

Reply:
Updated the comment.